src/styles/_catalog.scss

```
@@ -1,9 +1,10 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  place-items: center;
```

Use `justify-content: center` to center the grid, because now you are centering the element but not the grid itself

```suggestion
  justify-content: center;
```

---

Виправила!